README.md

```diff
@@ -58,181 +30,104 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
+# 📖 [Hello Microverse] <a name="about-project"></a>
 
-# 📖 [your_project_name] <a name="about-project"></a>
+> This is a HTML and CSS Project setup with Linters. 
 
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+**[Hello Microverse]** is the first project that new Microverse students have to do. In order to do complete this project linters have to be setup, to automatically check that code complies with HTML and CSS standards. 
```

Comment:
This is not the correct name for this project. Please make sure that you readme file correlates to your project.

Reply:
- [ ] Awesome review! But can you please state the reason(s) for the required changes? 👍